libs/ha/ha_common_libs_k8s.py

```diff
@@ -1296,6 +1308,23 @@ def simulate_disk_cvg_failure(self, node_obj, source, resource_status, resource_
             LOGGER.error("Please provide correct count for nodes")
             return False, node_id_list
         count = 0
+        user_val = False
+        if node_id and resource_id:
+            # If user provide node_id and resource_id
+            n_id = node_id
+            r_id = resource_id
+            user_val = True
+        elif node_id and resource_type == 'node':
+            # If user provide node_id
+            n_id = node_id
+            r_id = node_id
+            user_val = True
+        elif resource_id and resource_type == 'node':
+            # If user provide resource_id
+            n_id = resource_id
+            r_id = resource_id
+            user_val = True
+        specific_info = specific_info if specific_info else dict()
```

Comment:
Once above change is done, please remove this statement.

Reply:
Removed